product_docs/docs/edbcloud/beta/administering_cluster/02_postgres_access.mdx

```diff
@@ -20,13 +20,18 @@ Don't use the `edb_admin` user or the `edb_admin` database in your applications.
 
 ## One Database with One Application 
 
-For one database hosting a single application: 
+For one database hosting a single application, replacing app1 with your preferred user name: 
 
-1. Create a new database role. For example,
+1. Create a new database user. For example,
     ```
-    edb_admin=# create role app1 with password 'app1_pwd';
+    edb_admin=# create user app1 with password 'app1_pwd';
     ```
 
+1. Grant app1 to edb_admin
```

Comment:
Assign the `edb_admin` role to your new user. For example,

Reply:
@theadamwright, By resolving the conversation, did you mean to not accept the suggestion? How about we keep the initial step but add a period instead if my suggestion was not quite right?